geode-core/src/main/java/org/apache/geode/cache/client/internal/AutoConnectionSourceImpl.java

```diff
@@ -245,8 +245,6 @@ protected void updateLocatorInLocatorList(HostAddress locator) {
               locator.getSocketInetAddressNoLookup().getPort());
           HostAddress hostAddress = new HostAddress(changeLoc, locator.getHostName());
           newLocatorsList.add(hostAddress);
-          logger.info("updateLocatorInLocatorList changing locator list: loc form: " + locator
-              + " ,loc to: " + changeLoc);
         } else {
           newLocatorsList.add(tloc);
         }
```

Comment:
While you're in this code, you might consider changing the remaining log statement immediately after this block to correctly use Log4j parameters:
```
logger.info("updateLocatorInLocatorList locator list from: {} to: {}", 
    locatorList.getLocators(), newLocatorsList);
```
It's probably ok to improve the log message too if you really want to.

Reply:
Good point, I'll fix the other one to use log4j params correctly.
If someone is debugging the locator list changing might be significant information so I'll leave it in, maybe someone can worry about it later if it bothers them :P